docs/02-connect/04-eon_block_explorer.md

```diff
@@ -0,0 +1,584 @@
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-landing.png").default} alt="landing-page" width="900" height="700" />
+</p>
+
+**Note:** Use **Search** to query for specific transaction, address (wallet or contract), block, and token.
+
+
+# Key Features
+
+The key features of the Gobi Testnet Block Explorer are modularized in each of the panels in the main landing page. 
+
+## Daily Transaction
+
+The **Daily Transaction** dashboard provides a real-time view of the blockchain with various data points.
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-daily.png").default} alt="daily-transaction" width="900" height="700" />
+</p>
+
+The categories in the Daily Transaction dashboard includes:
+
+- A **graph view** of transactions processed during a time period. It includes metrics for tracking gas and transactions on a daily basis.
+- **Average block time** - This is the average time that a block is created.
+- **Total transactions** - The total number of transactions in a time period in the blockchain.
+- **Total blocks** - The total number of blocks in the blockchain.
+- **Wallet address** - The total number of wallet addresses associated with transactions in the blockchain.
+
+## Blocks
+
+The Block module provides key information about the block.
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-blocks.png").default} alt="blocks-module" width="900" height="700" />
+</p>
+
+Key information includes:
+
+- **Block number** - The number assigned to the block.
+- **Number of transactions** - The running number of transactions for the block.
+- **Number of bytes** - The amount of bytes for the block.
+- **Elapsed timer for the block** - Clock counting since block creation period.
+- **Miner (wallet) address** - The miner’s address. Click the address to Address Detail page.
+
+## Transactions
+
+The **Transaction** module provides key information about the transaction.
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-transactions.png").default} alt="transactions" width="900" height="700" />
+</p>
+
+Key information includes:
+
+- **Transaction Status** - Success or Error: (Awaiting internal transaction for reason)
+- **Transaction ID (hash)** - The unique identifier that validates the transaction.
+- **Address** - The wallet address associated with the transaction. 
+- **Transaction Fee** - The amount charged to process the transaction.
+- **Contract Address** - The contract address associated with the transaction. Click the address to **Address Detail** page.
+- **(Receiving) Address** - The receiving address associated with the transfer transaction. Click the address receiving asset to **Address Detail** page.
+
+## Gobi Testnet Block Explorer Features
+
+The Gobi Testnet Block Explorer gives you the ability to search the blockchain for transactions, addresses, and statistics, including the customized features of:
+
+- Backward Transfer
+- Forward Transfer
+- Fee Payment
+- Backward Transfer

```

Comment:
`Backward Transfer` is here twice. Also, we should keep these in the order of which they will show in the section below (if it's not being done already). I'm not sure where the **Backward Transfer** section is on the page. 

I would also suggest making Backward Transfer, Forward Transfer and Fee Payment sub-sections of this section so it appears like that in the sidebar navigation

Reply:
correction made